src/components/organisms/Navbar/Navbar/Navbar.md

```diff
@@ -1,27 +1,373 @@
 ### Summary
 
-`<Navbar>` renders the base navigation bar layout, creating **right**, **left**, and **center** to render the contents you like.
+By default [`<Navbar.LinksList />`](#/Components/Organisms/Navbar/NavbarLinksList) will render [`<Navbar.Dropdown />`](#/Components/Organisms/Navbar/NavbarDropdown) or [`<Navbar.Link />`](#/Components/Organisms/Navbar/NavbarLink) based on whether links have `children` or not.
 
-It's meant to be used with other navbar components like [`<Navbar.Link />`](#/Components/Organisms/Navbar/NavbarLink) and [`<Navbar.Dropdown />`](#/Components/Organisms/Navbar/NavbarDropdown)
+⚠️ In the examples below the menu will disappear when scrolling down and reappear when scrolling up
 
 ### Examples
 
-- White theme
+- Default theme
 
-```ts { "props": { "style": { "transform": "translate3d(0, 0, 0)", "backgroundColor": "#00B9A7", "border": "2px solid #efefef" } } }
-import { Navbar, Link } from '@zopauk/react-components';
+```ts { "props": { "style": { "transform": "translate3d(0, 0, 0)", "backgroundColor": "#00B9A7", "border": "2px solid #efefef", "overflow": "hidden" } } }
```

Comment:
To have an example close to the real life usage, it's probably better if you switch to white background and give it a bit of height as the examples get cut off